db/schema.rb

```diff
@@ -48,6 +48,7 @@
     t.datetime "reject_by_default_feedback_sent_at"
     t.datetime "offer_changed_at"
     t.datetime "last_public_update_at"
+    t.boolean "declined_by_default_at", default: false, null: false
```

Comment:
Doesn't the `t.boolean "declined_by_default", default: false, null: false` on line 36 serve the same purpose? 🤔

Reply:
🤦🏼‍♀️ I think somewhere between decline*d* and decline I missde the existing variable. Good catch, will remove.

Reply:
Removed. I think I got a bit confused with all the similar sounding field names here.